internal/core/workers/metricsreporter/metrics_reporter_worker.go

```diff
@@ -54,6 +55,7 @@ type MetricsReporterWorker struct {
 func NewMetricsReporterWorker(scheduler *entities.Scheduler, opts *workers.WorkerOptions) workers.Worker {
 	return &MetricsReporterWorker{
 		schedulerName:   scheduler.Name,
+		game:            scheduler.Game,
```

Comment:
Is there a specific reason to not give the entire scheduler here? Feels like for every new scheduler property we will change this struct and give scheduler.Something 🤔

Reply:
Maybe we were not including because we didn't need any other scheduler field... maybe it makes more sense to have the entire scheduler now